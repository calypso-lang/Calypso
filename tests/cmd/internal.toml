bin.name = "calypso"
args = "internal help"
