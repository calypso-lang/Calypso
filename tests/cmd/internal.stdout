calypso-internal 
Commands used for debugging Calypso's internals and implementation

USAGE:
    calypso internal <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    build-info    Show information about the environment where this executable was built
                      [aliases: bi, buildinfo]
    help          Print this message or the help of the given subcommand(s)
    panic         Intentionally panic in order to test out handling of ICEs (internal compiler
                      errors)
    unpretty      Run analyses on Calypso source files and return the result in an "unpretty"
                      format, e.g. AST (abstract syntax tree) or token list [aliases: up]
